### ISSUE TITLE
Add Rocketchat support

### ISSUE THREAD

**Issue by krakazyabra** [metalmatze/alertmanager-bot#82]
Hi! It will be great, if the bot can send messages to rocketchat. 
May be it is similar as Slack.

**Comment by metalmatze** [comment]
Yes, that would be another valid chat system. :slightly_smiling_face:

**Comment by mfominov** [comment]
or u can use
https://github.com/pavel-kazhavets/AlertmanagerRocketChat
